wiki/en/Installation-for-Linux.md

```diff
@@ -54,22 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If no instance of a JACK server is running, one is normally started automatically.
+
+Jamulus will attach to a running JACK instance by name, using the `JACK_DEFAULT_SERVER` environment variable or `default` if the variable is not set.
```

Comment:
So sometimes the JACK instance is named `default` ?

Reply:
Maybe. I don't know.
Anyone?

Reply:
Yes. I think so. At least Jamulus shows that.

Reply:
I will resolve this comment if anyone is ok as it is like that.